### ISSUE TITLE
[0.9.0-rc32] Unable to write any data via console / API: "database is required"

### ISSUE THREAD

**Issue by contentfree** [influxdb/influxdb#2834]
Brand new install of rc32. I have created a database `test`. Attempting to do the following results in a 400 with the error "database is required":

```
$ curl -XPOST 'http://myserver:8086/write' -d '
{
    "database": "test",
    "retentionPolicy": "default",
    "time": "2015-05-22T16:42:01.234Z",
    "precision": "ms",
    "points": [
        {
            "name": "testing",
            "tags": {
                "host": "server01",
                "region": "us-west"
            },
            "fields": {
                "value": 0.64
            }
        }
    ]
}'
```

Strangely, and maybe related, in the CLI, I see this:

```
> show databases
name: databases
---------------
name
test

> use test
Using database test
> show series
ERR: database not found: test
> show retention policies
ERR: error parsing query: found EOF, expected identifier at line 1, char 25
> show measurements
ERR: database not found: test
````

**Comment by toddboom** [comment]
Dave,

The field called `name` was renamed to `measurement` as of RC31. That
should make the writes work correctly, though I'm unsure about what's going
on in the second part. Is this just on a single node instance?

Todd

**Comment by contentfree** [comment]
Single node. 

Changing `name` to `measurement` made no difference. Same error. 

It looks like influxdb isn't creating anything in its data directory (which is configured to `/var/opt/influxdb/db`. Witness:

```
[user@myserver src]$ sudo service influxdb restart
influxd process was stopped [ OK ]
Starting the process influxd [ OK ]
influxd process was started [ OK ]
[user@myserver src]$ /opt/influxdb/versions/0.9.0-rc32/influx
Connected to http://localhost:8086 version FIXME
InfluxDB shell 0.9.0-rc32
> show databases
name: databases
---------------
name
test
statistics

> use statistics
Using database statistics
> show series
ERR: database not found: statistics
> exit
[user@myserver src]$ ls -al /var/opt/influxdb/db
total 8
drwxr-xr-x 2 influxdb influxdb 4096 Jun  8 21:58 .
drwxr-xr-x 5 influxdb influxdb 4096 Jun  8 21:58 ..
```

**Comment by jwilder** [comment]
The write also fails because there is leading whitespace in your curl `-d` arg between the `'` and the the `{` char.  #2835 should fix that, but if you remove the space\newline, it should write correctly.

**Comment by pbooth** [comment]
Im seeing the same behavior in an instance that was built today. I did find that renaming "name" as "measurement" meant that writes appeared to be doing something (or at least didn't return an error).

In my situation I was writing to a DB "mydb" and in the CLI had typed USE MYDB;
When I queried "show measurements;"  I got the error
ERR: database not found: MYDB

It hadn't occurred to me that database names might be case sensitive.
